### PR TITLE
Authentication additions

### PR DESCRIPTION
**Pull request by jrundquist**
Authentication now exists client side (although it is never to be trusted)

This branch also pulls in some of the filter panel and nav-dialog branches, although they will still have to have pull requests of their own. 

Additionally the tutorial image has been added (along with the .psd file if we need to change it further before launch)

Main branch has been pulled in as well due to some bug fixes on live that may affect our development.

### CODE REVIEW

File: lib/crawler/lib/schools/banner.js [hunk 1]
```diff
@@ -178,7 +201,7 @@ var banner = module.exports = everySchool.submodule('banner')
 							credits = ''+parseInt(credits[1],10)
 						}
 					}else{
-						console.log($sectionDetailsContainer.text(), credits);
+						//console.log($sectionDetailsContainer.text(), credits);
```
Comment (Gerg) [inline lib/crawler/lib/schools/banner.js]: Delete?

File: public/scripts/backbone-schedule/views/filter.coffee [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+			@render();
+
+		events:
+			'slide .slider': 'slideUpdate'
+
+		slideUpdate: ->
+			$slider = @$el.find('.slider')
+			value = $slider.slider('values')
+			console.log 'yay jquery-ui is working!', value
```
Comment (Gerg) [inline public/scripts/backbone-schedule/views/filter.coffee]: debug

File: public/scripts/backbone-schedule/views/filter.coffee [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+			# @$el.formatTime(value)
```
Comment (Gerg) [inline public/scripts/backbone-schedule/views/filter.coffee]: delete

File: public/scripts/backbone-schedule/views/filter.coffee [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+		render: ->
+			@$el.append @filterTemplate()
+			# 5-minute increments [0 - 24*60/5 == 0-288]         range starts at 7am - 7pm
+			@$el.find('.slider').slider
+				range: true
+				min: 72
+				max: 276 
+				values: [84, 228]
+			@$el.find('.time').text "7:00 am - 7:00 pm"
```
Comment (Gerg) [inline public/scripts/backbone-schedule/views/filter.coffee]: Can't this be done in the template?

File: public/styles/schedule/filter.less [hunk 5]
```diff
@@ -0,0 +1,107 @@
+@cs-blue: #3FA9F5;
+@sam-grey: #8a949e;
+
+
+.filter-panel {
+	padding: 8px 8px 8px 8px;
```
Comment (Gerg) [inline public/styles/schedule/filter.less:6]: padding: 8px;

File: public/scripts/backbone-schedule/views/filter.coffee (hunk 4 above)
Comment (jrundquist) [inline public/scripts/backbone-schedule/views/filter.coffee]: The slider initialization no, but the time text yes.

**Comment by jrundquist** [comment]
We also must make note of these debug statements as most came from @branon 's branch, so we may see them again when he pulls. Although hopefully git will see the changes and ignore them?

File: public/styles/schedule/filter.less [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  			margin-bottom: 15px;
+  			padding-left: 0.5em;
+  			padding-right: 0.5em;
+  		}
+  	}
+
+  	.days{
+  		text-align: center;
+  		margin: 10px;
+		.day {
```
Comment (Gerg) [inline public/styles/schedule/filter.less:40]: indent

File: public/styles/schedule/filter.less [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+  	.time {
+  		color: @cs-blue;
+  		margin-left: 10px;
+  	}
+
+  	/** Slider Stylings **/
+  	.slider {
+  		margin: 10px;
+  	}
+	.ui-slider {
```
Comment (Gerg) [inline public/styles/schedule/filter.less:54]: indention seems off

File: public/styles/schedule/framework.less [hunk 8]
```diff
@@ -14,7 +14,7 @@ body{
 .panel{
 	background: #fff;
 	border-radius: 4px;
-	box-shadow: 0 0 6px rgba(0,0,0,.7);
+	//box-shadow: 0 0 6px rgba(0,0,0,.7);
```
Comment (Gerg) [inline public/styles/schedule/framework.less:17]: delete?

File: public/styles/schedule/nav.less [hunk 9]
```diff
@@ -0,0 +1,5 @@
+@cs-blue: #3FA9F5;
```
Comment (Gerg) [inline public/styles/schedule/nav.less:1]: we should probably define this somewhere higher, so we don't have to keep redefining it

File: lib/crawler/lib/schools/banner.js (hunk 1 above)
Comment (jrundquist) [inline lib/crawler/lib/schools/banner.js]: Actually uncomment, but good catch.

**Comment by Gerg** [comment]
Other than a few nit picks looks good. #shipit

**Pull request merged**